app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md

```diff
@@ -85,10 +85,112 @@ The Becoming a Teacher team
 
 {% from "email/macro.njk" import appEmail %}
 {{ appEmail({
-  subject: "Course [Course name] (code) has been created",
+  subject: "Course [course_name] (course_code) has been published",
   content: template2
 }) }}
 
+## Email template: A course has been created
+
+{% set template6 %}
+Dear colleague,
+
+A new course has been created in Publish teacher training courses, and your organisation has been added as the accredited body for it.
+
+((Provider_name)) created ((course_name)) ((course_code)) at ((create_course_datetime)). The course has not yet been published on Find postgraduate teacher training. You’ll receive a notification if it is published.
+
+View the course at ((publish_url)).
```

Comment:
Where will this link to?

Reply:
It's a placholder variable(?), it goes to the courses as an accredited body screen.